src/scss/layout/_anchor-link.scss

```diff
@@ -0,0 +1,67 @@
+@use '~accoutrement/sass/tools';
+
+.anchor-link-wrapper {
+  display: inline-block;
+  position: relative;
+
+  h2 {
+    display: inline-block;
+    padding-left: var(--anchor-link-padding-l, var(--gutter));
+
+    @include tools.above('wide-page') {
+      --anchor-link-padding-l: 0;
+    }
+  }
+}
+
+.header-anchor {
+  --left-position-offset-default: -0.25em;
+  --top-position-offset-default: var(--gutter-plus);
+  left: var(--anchor-position-offset-l, var(--left-position-offset-default));
+  position: absolute;
+  top: var(--top-position-offset-default);
+  transform-origin: top center;
+  width: var(--gutter);
+
+  @include tools.above('wide-page') {
+    --anchor-position-offset-l: -1em;
+  }
+
+  svg {
+    stroke: var(--fill, currentColor);
+    stroke-width: 1.25;
+  }
+
+  &:visited {
+    --fill: var(--ccs-accent--bg1-fade);
+    --action: var(--ccs-accent--bg1-fade);
+  }
+
+  &:hover,
+  &:focus,
+  &:active {
+    --fill: var(--ccs-accent--fg2);
+  }
+}
+
+.anchor-clicked {
+  &::before {
+    @include tools.animate('fadein-out');
+    background: var(--feature);
+    border-radius: var(--radius);
+    color: var(--bg);
+    content: 'Link Copied';
+    font-size: var(--xxsmall);
+    left: var(--left-position-offset-default);
+    opacity: 0;
+    padding: var(--quarter-shim) var(--half-shim);
+    position: absolute;
+    top: calc(var(--top-position-offset-default) * -1);
+    white-space: nowrap;
+  }
+
+  svg {
+    transform: rotate(-45deg);
+    transition: tools.change('move');
```

Comment:
@stacyk @dvdherron When running the changes locally, I am seeing the same issue (content touching left viewport) so I don't think its because of the Netlify banner. I did notice there is a `overflow-x: hidden` style and when I toggle that, the viewport seems to be correct - could that be the culprit? 
<img width="697" alt="Screen Shot 2022-06-13 at 5 13 20 PM" src="https://user-images.githubusercontent.com/7397798/173387730-056372a8-288c-432c-932b-59d38f3712d6.png">

Reply:
@sanajaved7 @stacyk Hmm. I've this happen sometimes, but it doesn't happen consistently. . . I'll take another look to see if it's something we've done here that's causing it.

Reply:
and @stacyk I moved the transition to `header-anchor`

Reply:
@dvdherron when I turn my phone to get the close icon on that netlify banner, I am not seeing the same issue anymore so I am guessing we won't have this same problem when we test this on the live site.

Reply:
oops, I just read @sanajaved7's message again about testing locally ... I still this we can move forward with this and create a new card for the small screen anchor-landing positioning

Reply:
@stacyk @sanajaved7 Okay, after playing with this some more I think I've found a way where I can consistently make this overflow happen. I think it has something to do with the length of the section titles on each page. 

- Compare the ["Dependabot"](https://deploy-preview-206--oddleventy.netlify.app/2022/06/01/dependabot-single-pull-request/) article (longer headings) with, say, the "Support (Not) Unknown" article. 
- If you click and copy all the headings on the "Support" article, the overflow never happens. 
- If you click the longer heading in the Dependapot article ("Upgrade Node (JavaScript) Dependencies") the overflow happens and, depending on the browser, the page gets stuck in the overflow view and I think you have to reload the original link to the article to correct the layout. 

Can you all duplicate this?